### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by dminyaylo**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-flake8-annotations==2.9.1
-flake8-quotes==3.3.1
+flake8==6.0.0
+flake8-annotations==3.0.0
+flake8-quotes==3.3.2
 flake8-variables-names==0.0.5
-pep8-naming==0.13.2
-pytest==7.1.3
+pep8-naming==0.13.3
+pytest==7.3.1
```
Comment (Vasyl-Smutok) [inline requirements.txt:6]: don't change `requirements.txt`

File: app/cinema/hall.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self, movie_name: str,
+                      customers: list,
+                      cleaning_staff: object) -> None:
```
Comment (Vasyl-Smutok) [inline app/cinema/hall.py:7]:
```suggestion
    def movie_session(
            self, 
            movie_name: str,
            customers: list[???],
            cleaning_staff: Cleaner
    ) -> None:

```

File: app/main.py [hunk 3]
```diff
@@ -1,6 +1,38 @@
...
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list,
+                 hall_number: int,
+                 cleaner: str,
+                 movie: str) -> None:
+    customer_objects = [Customer(name=c["name"],
+                                 food=c["food"]) for c in customers]
```
Comment (Vasyl-Smutok) [inline app/main.py:12]:
```suggestion
    customer_objects = [
        Customer(name=c["name"], food=c["food"]) for c in customers
    ]
```

**Review by Vasyl-Smutok — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (dminyaylo) [reply requirements.txt:6]: if i dont change them, pytest and others doesn`t work

**Pull request closed without merging**